index.html

```diff
@@ -3587,6 +3587,9 @@ <h3>Parsing content</h3>
     To <dfn>parse records from bytes</dfn> given |bytes:byte sequence|,
     run these steps:
     <ol class=algorithm>
+      <li>
+        If the length of |bytes| is `0`, return `null` and abort these steps.
```

Comment:
Even if the length is >0, |bytes| may be a plain buffer that can not be parsed to get an NDEF message. From what I read, the current algo does not seem to be checking whether |bytes| is in the format of NDEF message or not...